example.js

```diff
@@ -0,0 +1,353 @@
+var mutate = require('xtend/mutable')
+var expose = require('choo-expose')
+var css = require('sheetify')
+var html = require('bel')
+var choo = require('./')
+
+css('todomvc-common/base.css')
+css('todomvc-app-css/index.css')
+
+if (module.parent) {
+  exports.todoStore = todoStore
+} else {
+  var app = choo()
+
+  if (process.env.NODE_ENV !== 'production') {
+    var persist = require('choo-persist')
+    var logger = require('choo-log')
+    app.use(persist())
+    app.use(logger())
+  }
+  app.use(expose())
+  app.use(todoStore())
+
+  app.route('/', mainView)
+  app.route('#active', mainView)
+  app.route('#completed', mainView)
+  app.mount('body')
+}
+
+function mainView (state, emit) {
+  emit('log:debug', 'Rendering main view')
+  return html`
+    <body>
+      <section class="todoapp">
+        ${Header(state, emit)}
+        ${TodoList(state, emit)}
+        ${Footer(state, emit)}
+      </section>
+      <footer class="info">
+        <p>Double-click to edit a todo</p>
+        <p>choo by <a href="https://yoshuawuyts.com/">Yoshua Wuyts</a></p>
+        <p>Created by <a href="http://shuheikagawa.com">Shuhei Kagawa</a></p>
+      </footer>
+    </body>
+  `
+}
+
+function todoStore () {
+  return function (state, emitter) {
+    var localState = state.todos
```

Comment:
What I'm gathering from this block of `localState` modifications is that the way you update state in choo `v5` is by modifying the global `state` object, which is passed down to "models" (things that are `use`d). That pattern is a bit unclear here because it _looks_ like we're modifying a local variable, and not actually persisting that globally anywhere. In fact, because of the way JavaScript variables work by reference with objects, we _are_ modifying a global variable, but you'd have to be familiar with that to pick up on what's going on here. My suggestion would be to make modifications to `state.todos.active = []` etc. directly for simplicity's sake.